client/src/pages/CreateProfilePage/CreateProfilePage.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
```

Comment:
why did y'all decide to make a copy of the MyProfile component for CreateProfilePage? it just seems a bit redundant to me since they essentially contain the same content and the edit/display mode and the save/done button can be determined by a prop.